study-builder/tenants/testboston/rules/link-admin-accounts.js

```diff
@@ -0,0 +1,119 @@
+// This rule will look for admin account and link it to the Google social account. Account link is
+// determined by using the email. This is only executed for the admin client and only if the
+// accounts are not already linked.
+//
+// This was derived from Auth0's rule template from here:
+// https://github.com/auth0/rules/blob/b381ef4a1d5df16c1716ea066f46cff87b88f7f1/src/rules/link-users-by-email-with-metadata.js
```

Comment:
It sounds like Auth0 used to provide this as a starting template when you create a new rule from the dashboard, but apparently they removed it (not sure why). I found their code in their git history.